edgexfoundry/edgex-taf
# fix: document for backward compatibility test

Document and execution file for backward compatibility test.
For more details [edgex-taf-pipeline #108](https://github.com/edgexfoundry/edgex-taf-pipelines/issues/108)

Signed-off-by: Cherry Wang <cherry@iotechsys.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions

See [run-bacward-compatibility-test-on-local.md](https://github.com/edgexfoundry/edgex-taf/compare/main...cherrycl:bct-docs?expand=1#diff-e588c33695a934fa11f7c38bb4f661422d7a547f1102302109218b2f75ed3e32)

## New Dependency Instructions (If applicable)
